corehq/apps/smsforms/migrations/0003_auto__chg_field_sqlxformssession_couch_id.py

```diff
@@ -0,0 +1,43 @@
+# encoding: utf-8
+import datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        # Changing field 'SQLXFormsSession.couch_id'
+        db.alter_column(u'smsforms_sqlxformssession', 'couch_id', self.gf('django.db.models.fields.CharField')(max_length=50))
```

Comment:
@snopoke do you know if this is also going to lock the table? (just adding a not null constraint)